apiTester/src/commonMain/kotlin/com/revenuecat/purchases/kmp/apitester/StoreProductAPI.kt

```diff
@@ -30,7 +30,7 @@ private class StoreProductAPI {
             val type: ProductType = type
             val price: Price = price
             val title: String = title
-            val description: String? = description
+            val description: String? = localizedDescription
```

Comment:
Hmm I'm still wondering whether it's worth deprecating `description` and just remove it once we release 1.0, so people using the SDK don't have to deal with these breaking changes right now...

Reply:
👍 Sounds good! We can keep it like this then